src/parser/SGTypes.ts

```diff
@@ -419,13 +434,21 @@ export class SGAst {
     }
 
     private updateScript(script: SGScript): SGScript {
+        let clone: SGScript;
```

Comment:
So do you want to keep that or change the logic to the node's `transpile` stage?

Reply:
I'm going to change this to node's `transpile` stage. I'm implementing the override of `transpileTags` concept that we discussed in Slack.